src/common/config/firebase.config.ts

```diff
@@ -6,27 +6,21 @@ import * as admin from "firebase-admin";
 export const firebaseConfig = registerAs<FirebaseConfig>(
   ConfigToken.GCP,
   () => {
+    // For local testing only: use service account for credentials.
+    // Ideally, we *should* be able to pass our own developer Application Default credentials
+    // into this instead of needing a service account. However, this turned out to be finnicky
+    // at the time, so we have opted to keep it like this for now.
+    // TODO: Figure out how to pass system application default credentials to this for local testing.
     if (process.env.GOOGLE_CERT) {
-      if (process.env.NODE_ENV && process.env.NODE_ENV === "production") {
-        const cert = JSON.parse(process.env.GOOGLE_CERT);
-        return {
-          credential: admin.credential.cert({
-            projectId: cert.project_id,
-            privateKey: cert.private_key,
-            clientEmail: cert.client_email,
-          }),
-          storageBucket: `${process.env.STORAGE_BUCKET}.appspot.com`,
-        };
-      } else {
-        // For local testing only:
-        // eslint-disable-next-line @typescript-eslint/no-var-requires
-        const serviceAccount = require(process.env.GOOGLE_CERT);
-        return {
-          credential: admin.credential.cert(serviceAccount),
-          storageBucket: `${process.env.STORAGE_BUCKET}.appspot.com`,
-        };
-      }
+      // eslint-disable-next-line @typescript-eslint/no-var-requires
+      const serviceAccount = require(process.env.GOOGLE_CERT);
```

Comment:
Is this so that we can get the service account credentials JSON file because the google_cert would be the path to the service account credentials?

Reply:
Yep, that's the idea. It's only used for local development since the Cloud Run instances are tied to a service account, so you just put the path to your credential file in your .env file.